HW5/src/js/task2.js

```diff
@@ -0,0 +1,77 @@
+let start = true,
+    game,
+    max = 5,
+    min = 0,
+    rendNumber,
+    resEnterNumber,
+    prize,
+    box = 0,
+    gold = 10,
+    silver = 5,
+    bronze = 2;
+
+    game = confirm("Do you want to play a game?");
+
+while (start) {
+    start = game;
+   if (game == false) {
+       alert("You did not become a millionaire, but can.");
+   }
+   if (game == true) {
```

Comment:
you can use just else

Reply:
if (game == false) {
       alert("You did not become a millionaire, but can.");
   } else {
  // here is your code ...
}